### DISCUSSION TITLE
Database constantly writing problem unsolved also moving to quiet SSD

### DISCUSSION THREAD

**Discussion by LordSpectre** [immich-app/immich#11135] [Q&A]
Hello, community, like for many other users I notice a constantly writing from postgress database. In my case, the annoying thing is that my home server is on my desk and I can hear noise every single second coming from my pool formed by 4 x Seagate Ironwolf Pro.
What I did to mitigate the noise was add a new virtual disk located on SSD and mount postgress there. The noise is now mitigated, but now **EVERY 5 SECONDS** I can still hear noise from disk.
What could be the process who still write on main disk and that I can move to SSD? Btw, iotop doesn't shown something interesting.

Here below some info from my environment:
**The OS that Immich Server is running on**
Ubuntu Server 24.04 (docker)

**Version of Immich Server**
v1.108.0

**Platform with the issue:**
Server

**Your docker-compose.yml content:**
>name: immich
>
>services:
>  immich-server:
>    container_name: immich_server
>    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
>    volumes:
>      - ${UPLOAD_LOCATION}:/usr/src/app/upload
>      - /etc/localtime:/etc/localtime:ro
>    env_file:
>      - .env
>    ports:
>      - 2283:3001
>    depends_on:
>      - redis
>      - database
>    restart: always
>
>  immich-machine-learning:
>    healthcheck:
>      disable: true
>    container_name: immich_machine_learning
>    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
>    volumes:
>      - model-cache:/cache
>    env_file:
>      - .env
>    restart: always
>
>  redis:
>    healthcheck:
>      disable: true
>    container_name: immich_redis
>    image: docker.io/redis:6.2-alpine@sha256:d6c2911ac51b289db208767581a5d154544f2b2fe4914ea5056443f62dc6e900
>    restart: always
>
>  database:
>    healthcheck:
>      disable: true
>    container_name: immich_postgres
>    image: docker.io/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
>    environment:
>      POSTGRES_PASSWORD: ${DB_PASSWORD}
>      POSTGRES_USER: ${DB_USERNAME}
>      POSTGRES_DB: ${DB_DATABASE_NAME}
>      POSTGRES_INITDB_ARGS: '--data-checksums'
>    volumes:
>      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
>    restart: always
>
>volumes:
>  model-cache:

```
**Your .env content:**
# The location where your uploaded files are stored
UPLOAD_LOCATION=./library
# The location where your database files are stored
DB_DATA_LOCATION=./postgres
# The Immich version to use.
IMMICH_VERSION=release
# PostgreSQL password. Change it to a secure one.
DB_PASSWORD=********

# The values below this line do not need to be changed
#######################################################
DB_USERNAME=postgres
DB_DATABASE_NAME=immich
```

**Comment by zackpollard** [comment]
This issue was resolved in 1.108.0, if you're still experiencing this, it may be from something other than Immich. You can use inotify to monitor the disks that are spinning up and detect what files are being written to/accessed every 5 seconds.

**Reply by LordSpectre** [reply]
Oh that's a good news, I didn't know it was solved in latest version. So now I have to point my finger elsewhere.
Most probably is Ubuntu server, so now I have to reinstall it on SSD and then use a second disk (NON SSD) for library and database.
Do you know if there is an easy way to migrate everything? I really don't want to re-import the whole things.

**Reply by bo0tzz** [reply]
I'd recommend actually investigating what the cause is before just blanket reinstalling everything. I doubt ubuntu server is the reason for this.

**Reply by LordSpectre** [reply]
That's reasonable, but most probably I'll take more time to investigate other than re-install everything on SSD disk.
The only thing I missed now is how to migrate library between VMs
